src/style-stars--1.css

```diff
@@ -0,0 +1,3 @@
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(images/star-active.svg);
```

Comment:
Avoid using tag names for styling elements. It's better to use classes to apply styles to your elements, as this allows for more flexibility and better performance. Instead of selecting `.stars__star:nth-child(1)`, consider giving the first child a unique class name and style it accordingly.